packages/react/src/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,165 @@
+import React, { useState, useRef } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { useId } from 'react-id-generator';
+import { useDidUpdate } from '../../index';
+import TabPanel from './TabPanel';
+import Tab from './Tab';
+
+const [left, right, home, end] = [37, 39, 36, 35];
+
+interface TargetElement extends EventTarget {
+  id?: string;
+}
+
+interface TabsProps {
+  value: number;
```

Comment:
`value` implies it could be anything, so having a more specific name would better indicate its usage.

Something like `activeTabIndex` would make it more clear. We could also make it optional, so that by default it defaults to the first tab.